data/validateQuestionLink.js

```diff
@@ -0,0 +1,80 @@
+import { questions } from "./question-data.js";
+import fs from "fs/promises";
+import path from "path";
+
+// Run this script and check logs/broken-links-logs.csv for broken links
+
+const STATUS_SUCCESS = 200;
```

Comment:
Let's keep the naming conventions consistent and use camelCase.

Reply:
fixed